docs/source/algorithms/HB3AIntegrateDetectorPeaks-v1.rst

```diff
@@ -14,15 +14,36 @@ reduction workflow, using :ref:`HB3AAdjustSampleNorm
 <algm-HB3AAdjustSampleNorm>` with `OutputType=Detector` which can be
 seen below in the example usage.
 
-This will reduced the input workspace using the region of interest
+This will reduce the input workspace using the region of interest
 provided to create a 1-dimension workspace with an axis that is the
-detector scan, either omega or chi in degrees. This reduced workspace
-is then fitted using :ref:`Fit <algm-Fit>` with a :ref:`flat
-background <func-FlatBackground>` and a :ref:`Gaussian
-<func-Gaussian>`, then the area of the Gaussian is used as the peak
-intensity and added to the output workspace. An optionally fitting
-range of the scan axis can be provided with `StartX` and `EndX` in
-degrees.
+detector scan, either omega or chi in degrees.
+
+There are three different methods for integrating the input workspace:
+simple counts summation, simple counts summation with fitted background,
+and a fitted model, which are described in detail at
+:ref:`HFIR Single Crystal Reduction <interface-HFIR-Single-Crystal-Reduction>`.
```

Comment:
Can you copy the equations into this algorithms docs, I believe that interface is going to be replaced at some point.

Reply:
> In the current way that new peaks methods are created, they will be using the first goniometer position in the scan (set by HB3AAdjustSampleNorm) which will mean that the q-lab vector is slightly wrong and then if they are saved with direction cosines they will be slightly wrong. In my first implementation, I used the fitted peak centre to get the correct goniometer value for the scan-axis which might be overkill. This isn't an option if you don't fit the peak, I would say using the centre or average value of the goniometer would have to be close enough? They try to scan the peaks so that it's in the centre of the scan.
> 
> We could just do `SetGoniometer(Workspace=__tmp_pw, Axis0='omega,0,1,0,-1', Axis1='chi,0,0,1,-1', Axis2='phi,0,1,0,-1')` for every integration method which will set the goniometer to the average of the scan.

Only the counts-only method doesn't do the fitting. I believe averaging the scan should be sufficient, however, since it really only effects the direction cosines and the range of angles aren't that large. Generally, the peak is centered in the scan range so that should be a good approximation.